templates/default/stunnel.conf.erb

```diff
@@ -63,6 +63,11 @@ debug = <%= node['stunnel']['debug'] %>
 output = <%= node['stunnel']['output'] %>
 <% end -%>
 
+; Enable/disable FIPS mode
+<% unless node['stunnel']['fips'].nil? %>
```

Comment:
With that detail I would make a note about this option either in a comment for that attribute or in the README about that attribute. Considering it was never included before, leaving it out via nil won't exactly be a breaking change for some since it was never there before.